babel/build.gradle.kts

```diff
@@ -52,6 +52,14 @@ val javaCCMain by tasks.registering(org.apache.calcite.buildtools.javacc.JavaCCT
     packageName.set("org.apache.calcite.sql.parser.babel")
 }
 
+tasks.autostyleJavaCheck {
+    dependsOn(javaCCMain)
+}
+
+tasks.checkstyleMain {
+    dependsOn(javaCCMain)
+}
```

Comment:
This would break in case `-PskipCheckstyle` or `-PskipAutostyle`.
Something like https://github.com/apache/jmeter/blob/0938f50e15cc62e2de645b823def881364e64b78/src/core/build.gradle.kts#L145-L160 might do.

Reply:
@vlsi  Please help review again. Thanks very much.

Reply:
> This would break in case `-PskipCheckstyle` or `-PskipAutostyle`. Something like https://github.com/apache/jmeter/blob/0938f50e15cc62e2de645b823def881364e64b78/src/core/build.gradle.kts#L145-L160 might do.

done.